README.md

```diff
@@ -177,7 +177,15 @@ teracy-dev-certs:
     - "%{node_hostname_prefix}.%{node_domain_affix}"
 ```
 
-For example:
+`ansible` has 2 modes: 
+
+- The `guest` mode (default) which is automatically installed in VM by vagrant. Its certificate is valid in 2000 days
```

Comment:
The `guest` mode (default) which is automatically installed in the VM machine by vagrant.